app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,59 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.routes.Response;
+import com.codesoom.assignment.routes.Tasks;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.stream.Collectors;
+
+/**
+ * HTTP handler.
+ */
+public class DemoHttpHandler implements HttpHandler {
+    private static final String HEALTH_CHECK_PATH = "/";
+    private static final String TASKS_PATH = "/tasks";
+
+    private HttpExchange exchange;
+
+    /**
+     * Get request body.
+     *
+     * @return request body.
+     */
```

Comment:
이렇게 하는 것으로 충분합니다.
```suggestion
    /**
     * Returns the request body.
     */
```
한국어로 적는 것도 좋습니다.
```suggestion
    /**
     * HTTP 요청 본문을 리턴합니다.
     */
```